### DISCUSSION TITLE
Does it use codemirror or monaco ?

### DISCUSSION THREAD

**Discussion by softmarshmallow** [codesandbox/codesandbox-client#5495] [Q&A] [answered]
'Cause i see both dependencies, and the editor itself seems to be using alot from vscode. It sandbox using codemirror? and if then, Why?

**Comment by sbinlondon** [comment, marked as answer]
Hi @softmarshmallow, this has been answered [here](https://github.com/codesandbox/codesandbox-client/discussions/5372) 😊
